src/nativestore/RelationBlock.cpp

```diff
@@ -347,7 +348,7 @@ RelationBlock* RelationBlock::getCentral(unsigned int address) {
     if (address == 0) {
         return NULL;
     } else if (address % RelationBlock::BLOCK_SIZE != 0) {
-        throw "Exception: Invalid relation block address !!\n received address = " + address;
+        throw std::logic_error("Exception: Invalid relation block address !!\n received address = " + address);
```

Comment:
Rather than throwing this error shall we log it?

Reply:
Changed to log error